python/lookout/sdk/pb.py

```diff
@@ -0,0 +1,13 @@
+"""
+Re-exporting all grpc related classes/functions for cleaner API
+"""
+
+from .service_analyzer_pb2_grpc import \
+    AnalyzerServicer, add_AnalyzerServicer_to_server as add_analyzer_to_server
```

Comment:
`AnalyzerServicer ` does not need to be imported here. Sorry - I should have spotted this earlier!

Reply:
True. Leftover after copy-past. Thank you!